pkgs/by-name/xi/xiasl/package.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeDesktopItem
+, copyDesktopItems
+, libsForQt5
+, imagemagick
+, acpica-tools
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xiasl";
+  version = "1.1.67";
+
+  src = fetchFromGitHub {
+    owner = "ic005k";
+    repo = "Xiasl";
+    rev = finalAttrs.version;
+    hash = "sha256-6RJeNLgeFchCjYLcwKgGtV0dLyHO7gYpIpo/3BBh/wM=";
+  };
+
+  buildInputs = [
+    libsForQt5.qtbase
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    libsForQt5.wrapQtAppsHook
+    libsForQt5.qmake
+  ];
+
+  postPatch = ''
+    rm libqscintilla2_qt5.so
+    ln -s '${lib.getLib libsForQt5.qscintilla}/lib/libqscintilla2_qt5.so' .
+
+    substituteInPlace Xiasl.pro \
+      --replace-warn '/opt/$''${TARGET}' "$out"
+
+    substituteInPlace *.cpp \
+      --replace-quiet 'appInfo.filePath() + "/iasl"' '"${acpica-tools}/bin/iasl"' \
+      --replace-quiet 'appInfo.filePath() + "/acpidump"' '"${acpica-tools}/bin/acpidump"' \
+  '';
+
+  postInstall = ''
+    install -Dm644 icon.png "$out/share/pixmaps/xiasl.png"
+  '';
+
+  preFixup = ''
+    patchelf --add-rpath '${lib.makeLibraryPath [ libsForQt5.qscintilla ]}' "$out/bin/Xiasl"
```

Comment:
Maybe use `autoPatchelfHook` with `runtimeDependencies`?

Reply:
I'd rather do this one library manually as it's required due to a quirk in the build system. `autoPatchelfHook` could let future mistakes slip through unnoticed.

Reply:
> autoPatchelfHook could let future mistakes slip through unnoticed.

But how? `runtimeDependencies` are always added in the same way you used.